modules/assim.sequential/tests/Rcheck_reference.log

```diff
@@ -602,22 +588,10 @@ sda.enkf: no visible global function definition for
 sda.enkf: no visible global function definition for ‘logger.severe’
 sda.enkf: no visible global function definition for ‘cov’
 sda.enkf: no visible global function definition for ‘rmvnorm’
-sda.enkf.multisite: no visible binding for global variable
-  ‘multiprocess’
-sda.enkf.multisite: no visible global function definition for ‘map’
-sda.enkf.multisite: no visible global function definition for ‘map_dfr’
-sda.enkf.multisite: no visible global function definition for ‘tail’
 sda.enkf.multisite: no visible binding for global variable
   ‘ensemble.samples’
-sda.enkf.multisite : <anonymous>: no visible global function definition
-  for ‘input.ens.gen’
-sda.enkf.multisite : <anonymous>: no visible global function definition
-  for ‘write.ensemble.configs’
-sda.enkf.multisite : <anonymous>: no visible binding for global
-  variable ‘ensemble.samples’
-sda.enkf.multisite: no visible global function definition for
-  ‘setNames’
-sda.enkf.multisite: no visible global function definition for ‘map_dfc’
+sda.enkf.multisite: no visible global function definition for ‘write.ensemble.configs’
+sda.enkf.multisite: no visible global function definition for ‘input.ens.gen’
```

Comment:
do these two really need to be added? Above it looked like you provided the namespace for both

Reply:
Those two should be deleted.